File: Framework.Tests/ReceivableServices/AccountReceivable.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Framework.Data_Manipulation;
using Framework.Domain;

namespace Framework.Tests.ReceivableServices
{
    public class AccountReceivable: DomainObject
    {
        public AccountReceivable(IBaseMapper mapper)
            : base(mapper)
        {
        }

        [IdentityField]
        public string Number { get; set; }

        public string CustomerNumber { get; set; }

        public string Description { get; set; }
    }
}

File: Android/app/src/test/java/com/peaa/ReceivableServices/AccountReceivable.java
package com.peaa.ReceivableServices;

import com.peaa.DataManipulation.BaseMapperInterfaces.IBaseMapper;
import com.peaa.Domain.DomainObject;
import com.peaa.IdentityFieldAnnotation;

/**
 * Created by aiko on 4/8/17.
 */

public class AccountReceivable extends DomainObject {

    @IdentityFieldAnnotation
    public String Number;

    public String CustomerNumber ;

    public String Description;

    public AccountReceivable(String customerNumber, String number, IBaseMapper mapper)
    {
        super(mapper);

        CustomerNumber = customerNumber;
        Number = number;
    }
}
